CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ add_dependencies(rdkit_base catch)
 
 option(RDK_BUILD_SWIG_WRAPPERS "build the SWIG wrappers" OFF )
 option(RDK_BUILD_PYTHON_WRAPPERS "build the standard python wrappers" ON )
+option(RDK_INSTALL_PYTHON_PACKAGE "install Python wrappers via pip to the selected python" ON )
```

Comment:
This should be `OFF`, since it's not likely to be part of the standard build process for people at the moment

Reply:
Well, no, see what it does here:

https://github.com/rdkit/rdkit/pull/2690/files#diff-af3b638bc2a3e6c650974192a53c7291R300

It does exactly the same as old version did -- install Python bindings to `CMAKE_INSTALL_PREFIX/DESTDIR`.
Actually, I would argue that this option can be removed at all, making it the default behavior.